core/config/project_settings.cpp

```diff
@@ -148,7 +148,7 @@ const PackedStringArray ProjectSettings::_trim_to_supported_features(const Packe
 #endif // TOOLS_ENABLED
 
 String ProjectSettings::localize_path(const String &p_path) const {
-	String path = p_path.simplify_path();
+	String path = ResourceUID::ensure_path(p_path).simplify_path();
```

Comment:
The method expects a global path, so this should be unnecessary 🤔 (see `globalize_path()`)

Reply:
Here p_path is an uid, I can directly do the conversion here too https://github.com/godotengine/godot/blob/691d8bee2bdc316da32c7137783864a25c1ad674/main/main.cpp#L4190-L4191 and localize_path() is called here, https://github.com/godotengine/godot/blob/691d8bee2bdc316da32c7137783864a25c1ad674/editor/editor_node.cpp#L3982

Reply:
Then the path should be ensured in `load_scene()`.

Reply:
Initially, I was using that approach, but I switched to this method to avoid requiring conversion for any other calls to localize_path.